packages/dotcom-ui-header/src/components/drawer/topLevelPartials.tsx

```diff
@@ -30,12 +30,8 @@ const Drawer = (props: THeaderProps) => {
       data-trackable="drawer"
       data-trackable-terminate>
       <div className="o-header__drawer-inner">
-        <DrawerTools />
+        <DrawerTools {...editions}/>
```

Comment:
I see inconsistency in spacing between props and component closing tag. You way need to format your code

```tsx
<DrawerTools {...editions}/>
```

Reply:
done